packages/dialtone-mcp-server/README.md

```diff
@@ -0,0 +1,176 @@
+# Dialtone MCP Server
+
+A [Model Context Protocol (MCP)](https://modelcontextprotocol.io) server that provides AI assistants with search access to Dialtone's design system: utility classes, design tokens, and Vue components.
+
+## Quick Start
+
+### 1. Build
+
+From the monorepo root:
+
+```bash
+pnpm install
+pnpm nx run dialtone-mcp-server:build
+```
+
+### 2. Test Interactively
+
+Try out search queries (must be run from the package directory):
+
+```bash
+# Navigate to the package directory first!
+cd packages/dialtone-mcp-server
+
+# Then run the interactive tool
+pnpm run interactive
+```
+
+Or use node directly from anywhere:
+```bash
+node packages/dialtone-mcp-server/interactive-search.js
+```
+
+**Query Format:** Use keywords only, not questions.
+- ✓ Good: `padding 8px`, `color primary`, `button`
```

Comment:
haha, funny that my tests were like how Nina did... doing questions 🤪

`Hey, could you give me the token for font size 15px please? tyvm 🫰"